backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/Disk.java

```diff
@@ -31,6 +33,9 @@ public abstract class Disk extends BaseDisk {
     private boolean plugged;
     private String logicalName;
 
+    @TransientField
+    private transient Guid parentDiskId;
```

Comment:
```suggestion
    private Guid parentDiskId;
```
no need for the transient keyword, it's more about declaration than making it a transient field